ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml

```diff
@@ -3,4 +3,4 @@
     jcr:primaryType="cq:ClientLibraryFolder"
     allowProxy="{Boolean}true"
     categories="[hclecomm.base]"
-    embed="[core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,core.wcm.components.image.v2,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,hclecomm.grid,hclecomm.title,hclecomm.text,hclecomm.button,hclecomm.accordion,hclecomm.image,hclecomm.teaser,hclecomm.carousel,hclecomm.tabs,hclecomm.footerContainer,hclecomm.navigation,hclecomm.search,hclecomm.minicart,hclecomm.productdetails,hclecomm.cartProductDisplay,hclecomm.productcart,hclecomm.productfilter,hclecomm.checkoutform,hclecomm.signin,hclecomm.signup,hclecomm.checkoutcarttotal,hclecomm.confirmation,hclecomm.backtotop,hclecomm.container,hclecomm.list,hclecomm.languagenavigation,hclecomm.newsletter,hclecomm.wishlist,hclecomm.wishlistitem,hclecomm.contactus,hclecomm.form.text,hclecomm.form.button,hclecomm.recaptcha,hclecomm.orders,hclecomm.youraddress,hclecomm.loginDetail,hclecomm.ratings]"/>
+    embed="[core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,core.wcm.components.image.v2,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,hclecomm.grid,hclecomm.title,hclecomm.text,hclecomm.button,hclecomm.accordion,hclecomm.image,hclecomm.teaser,hclecomm.carousel,hclecomm.tabs,hclecomm.footerContainer,hclecomm.navigation,hclecomm.search,hclecomm.minicart,hclecomm.productdetails,hclecomm.cartProductDisplay,hclecomm.productcart,hclecomm.productfilter,hclecomm.checkoutform,hclecomm.signin,hclecomm.signup,hclecomm.checkoutcarttotal,hclecomm.confirmation,hclecomm.backtotop,hclecomm.container,hclecomm.list,hclecomm.languagenavigation,hclecomm.newsletter,hclecomm.wishlist,hclecomm.wishlistitem,hclecomm.contactus,hclecomm.form.text,hclecomm.form.button,hclecomm.recaptcha,hclecomm.orders,hclecomm.youraddress,hclecomm.loginDetail,hclecomm.ratings,hclecomm.chatbot]"/>
```

Comment:
@VikramGoud1576 : Since chatbot is a external tool , pl do not include the client-libs from here for this as an exception .

Add clientlib though the component HTML